src/features/cartographie/components/lieux-mediation-numerique-details/impression-choice-modal/impression-choice-modal.component.html

```diff
@@ -1,15 +1,15 @@
 <app-modal #modal heading="Imprimer" icon="ri-printer-line" bgBody="light" bgFooter="light">
   <div class="text-center">
-    <h5 class="h5 fw-bold text-primary">Vous êtes médiateur numérique ?</h5>
-    <div class="my-3">
+    <h5 class="h5 fw-bold text-primary mt-5">Vous êtes médiateur numérique ?</h5>
+    <div class="my-2 mb-4">
```

Comment:
```suggestion
    <div class="mt-2 mb-4">
```
`my-*` c'est une marge en haut en en bas. Si tu ajoutes `mb-4`, tu n'as plus besoin de la marge en bas à 2, tu peux donc remplacer par `mt-2` qui s'occuê juste de la marge en haut